src/libraries/PendingLib.sol

```diff
@@ -34,22 +34,21 @@ library PendingLib {
     function update(PendingUint192 storage pending, uint192 newValue, uint256 timelock) internal {
         pending.value = newValue;
         // Safe "unchecked" cast because timelock <= MAX_TIMELOCK.
-        pending.validAt = uint48(block.timestamp) + uint48(timelock);
+        pending.validAt = uint56(block.timestamp + timelock);
     }
 
     /// @dev Updates `pending`'s value to `newValue` and its corresponding `validAt` timestamp.
     /// @dev Assumes `timelock` <= `MAX_TIMELOCK`.
     function update(PendingAddress storage pending, address newValue, uint256 timelock) internal {
         pending.value = newValue;
         // Safe "unchecked" cast because timelock <= MAX_TIMELOCK.
-        pending.validAt = uint48(block.timestamp) + uint48(timelock);
+        pending.validAt = uint56(block.timestamp + timelock);
     }
 
     /// @dev Marks the market associated to the given config as removable starting `timelock` seconds in the future.
     /// @dev Assumes `timelock` <= `MAX_TIMELOCK`.
     function disable(MarketConfig storage config, uint256 timelock) internal {
-        config.cap = 0;
```

Comment:
I would keep it

Reply:
Can we continue the discussion here: https://github.com/morpho-org/metamorpho/pull/312#discussion_r1386189685 and can you argue?